conf/upf.json

```diff
@@ -88,6 +92,10 @@
     "" : "notify_sockaddr: /tmp/notifycp",
     "" : "endmarker_sockaddr: /tmp/pfcpport",
 
+    "": "Whether to enable UPF HeartBeatTimer feature",
+    "enable_hbTimer": false,
+    "": "heart_beat_interval: 5000",
```

Comment:
@clstefi This too will be needed for other requests right?

Reply:
also change resp_timeout and heart_beat_interval values to include the unit and use duration parse https://pkg.go.dev/time#ParseDuration